app/build.gradle.kts

```diff
@@ -102,6 +107,7 @@ dependencies {
     implementation(Libs.NAVIGATION_FRAGMENT_KTX)
     // Hilt
     implementation(Libs.HILT)
+    implementation(Libs.FIREBASE_AUTH)
```

Comment:
Why does it in between hilt deps?

Reply:
Mistakenly Placed it over there. Moved it to FireBase section.